cyberglove/launch/cyberglove.launch

```diff
@@ -2,11 +2,14 @@
   <arg name="serial_port" default="/dev/ttyUSB0"/>
   <arg name="calibration" default="$(find sr_cyberglove_config)/calibrations/cyberglove.cal"/>
   <arg name="version" default="2"/>
+  <arg name="protocol" default="8bit"/>
+  <!-- Activate internal cybeglove data filtering -->
+  <arg name="filter" default="false"/>
```

Comment:
true might be a better default for now? It was true before. Once it has been tested more extensively, then we can set it to true by default?